README.md

```diff
@@ -21,7 +21,8 @@ Any dataset specific notes on lexibank decisions/mapping choices etc go in here.
 ![Concepticon: 73%](https://img.shields.io/badge/Concepticon-73%25-yellow.svg "Concepticon: 73%")
 ![Source: 100%](https://img.shields.io/badge/Source-100%25-brightgreen.svg "Source: 100%")
 
-- **Varieties:** 128
-- **Concepts:** 1,523
-- **Lexemes:** 157,589
-- **Synonymy:** 1.36
\ No newline at end of file
+- **Varieties:** 127
+- **Concepts:** 1,581
+- **Lexemes:** 130,556
```

Comment:
Is the decrease of lexemes accounted for?

Reply:
There is also a number of problematic splits, e.g.:

- Tuveta-962_oilfromplants-1 `ngayo 1a/2a` `ngayo 1a`
- Tuveta-962_oilfromplants-2 `ngayo 1a/2a` `2a`

And other numbers in the forms:

- Tuveta-183_beard-1 `nyami 9`

Sorry if this was discussed elsewhere.